### DISCUSSION TITLE
Please add CONFIG_NET_SCH_FQ_CODEL=m to buildroot-external/kernel/docker.config

### DISCUSSION THREAD

**Discussion by unclehack** [home-assistant/operating-system#2202] [Ideas]
This will enable FQ_CODEL.
```
diff --git a/buildroot-external/kernel/docker.config b/buildroot-external/kernel/docker.config
index 40b07857..4c133bbc 100644
--- a/buildroot-external/kernel/docker.config
+++ b/buildroot-external/kernel/docker.config
@@ -3,6 +3,7 @@ CONFIG_CFQ_GROUP_IOSCHED=y
 CONFIG_CFS_BANDWIDTH=y
 CONFIG_FAIR_GROUP_SCHED=y
 CONFIG_NET_SCHED=y
+CONFIG_NET_SCH_FQ_CODEL=m
 # CONFIG_RT_GROUP_SCHED is not set

 CONFIG_CGROUPS=y
```

**Comment by unclehack** [comment]
@agners: I've started this discussion thread.

**Reply by agners** [reply]
Does this enable FQ Codel by default or is something more required? Do you have an integration/add-on which needs that?

**Reply by agners** [reply]
Hm, I see systemd defaults to fq_codel. On some platform we seem to be using it already (through board specific configs, e.g. RPi).

Yeah I think its sensible to enable that by default. I'd say it should go into `buildroot-external/kernel/hassos.config` even, since it's enabled by systemd.

**Reply by unclehack** [reply]
Yes, systemd enables it by default when the kernel module is present. Some boards seem to have it enabled in their configs.

**Reply by unclehack** [reply]
It's not required by anything at all. It's just a sane systemd default we should use.

**Reply by agners** [reply]
Adding the config with https://github.com/home-assistant/operating-system/pull/2203

**Reply by unclehack** [reply]
Will this be included in the 9.4 release? Will there be a 9.4 release or will it just be 10.0?

**Reply by agners** [reply]
Yes, it will be in the next 9.x release (if we do one, but that is almost guaranteed as 10.x is still some months out).
